README.md

```diff
@@ -6,15 +6,17 @@
 </p>
 <br>
 
-A white paper specifying an über-shader that aims to provide a material representation capable of accurately modeling the vast majority of materials used in practical visual effects and feature animation productions.
 
-OpenPBR is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
+OpenPBR Surface is a specification of a surface shading model intended as a standard for computer graphics. It aims to provide a material representation capable of accurately modeling the vast majority of CG materials used in practical visual effects and feature animation productions.
 
+OpenPBR Surface is an open standard hosted by the [Academy Software Foundation](https://www.aswf.io/) (ASWF), and is organized as a subproject of [MaterialX](https://materialx.org/).
 
-### Resources
+### Specification
 
-* **[Specification](https://academysoftwarefoundation.github.io/OpenPBR/)**
+* **[White paper](https://academysoftwarefoundation.github.io/OpenPBR/)**
+* **[Parameter reference](https://academysoftwarefoundation.github.io/OpenPBR/#parameterreference)**
 * **[Reference implementation](reference/open_pbr_surface.mtlx)** – written in [MaterialX](https://materialx.org/)
+* **[BibTeX citation](citation.bib)**
```

Comment:
👍
Minor remark: I would have named it `openpbr.bib`, as a convenience for people downloading the file.

Reply:
Done in 5244f958157f855e51980e8d4098e5e87c51cfd3